app/presenters/content_item/image.rb

```diff
@@ -0,0 +1,18 @@
+module ContentItem
+  module Image
+    def image
+      content_item["details"].dig("image") || default_news_image || placeholder_image
```

Comment:
Minor - but since we're moving this code around how do you feel about changing `content_item["details"].dig("image")` to `content_item.dig("details", "image")` there's another one below too.

I don't really know why someone did that originally, seems a bit weird to do a lookup then a dig :-/